### PR TITLE
Hello World Anton

### LINKED ISSUE #22: Hello World Anton
- create a new entity in the commons package which is called `<your-name>Pet` (exp: `BobsPet`)
- create a backend service which serves this entity (save the entity in memory, simple variable)
  - [Docs](https://docs.feathersjs.com/api/services.html)
- create a vue component which
  - triggers a call to your new backend service (for example by pressing a button) and creates an entity
  - shows a list of your entities (initial find method call) and adds newly created entities [`created` event](https://docs.feathersjs.com/api/events.html) to that list

### PR DESCRIPTION
**Pull request by anbraten**
closes #22

### CODE REVIEW

**Review by JanEberlein — APPROVED** [review]
Looks good ✔️

File: packages/app/src/components/HelloWorld.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    const antonsPets = ref<AntonsPet[]>([]);
+
+    onMounted(async () => {
+      const response = await feathers.service('antons-pets').find();
+      antonsPets.value = response.data;
+    });
+
+    const spawnAPet = async () => {
+      const newAntonsPet: AntonsPet = {
+        name: `Antons geile Maus geboren am ${Date.now()}`,
```
Comment (julianhahn) [inline packages/app/src/components/HelloWorld.vue:43]: please remove / rewrite this for presentation purposes

File: packages/commons/src/antons-pet.class.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default class AntonsPet {
+  name: string;
+
+  constructor(name: string) {
```
Comment (julianhahn) [inline packages/commons/src/antons-pet.class.ts:4]: maybe remove the constructor if we still don't use it.

**Review by julianhahn — CHANGES_REQUESTED** [review]

File: packages/app/src/components/HelloWorld.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/components/HelloWorld.vue]: resolved by ca477b4e180b8e15a3c15cb7e77ababe91d37c86

File: packages/commons/src/antons-pet.class.ts (hunk 2 above)
Comment (anbraten) [reply packages/commons/src/antons-pet.class.ts]: resolved by cefce26b6eaabb24c00da569b8f89a7fa9dcb5ab

File: packages/app/src/components/HelloWorld.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/components/HelloWorld.vue]: @julianhahn Is that fine for you?

Comment (julianhahn) [reply packages/app/src/components/HelloWorld.vue]: looking good now 🎉

**Review by julianhahn — APPROVED** [review]

**Pull request merged**